packages/flutter/test/material/tabs_test.dart

```diff
@@ -4183,6 +4183,40 @@ void main() {
 
     semantics.dispose();
   });
+
+  testWidgets("TabBar accepts 'splashBorderRadius' that gives 'borderRadius' to InkWell", (WidgetTester tester) async {
+    const Color _hoverColor = Color(0xfff44336);
+    const double _radius = 20;
+    await tester.pumpWidget(boilerplate(
+        child: DefaultTabController(
+            length: 1,
+            child: TabBar(
+                overlayColor: MaterialStateProperty.resolveWith<Color>(
+                  (Set<MaterialState> states) {
+                    if (states.contains(MaterialState.hovered)) {
+                      return _hoverColor;
+                    }
+                    return Colors.black54;
+                  },
+                ),
+                splashBorderRadius: BorderRadius.circular(_radius),
+                tabs: const <Widget>[
+                  Tab(
+                    child: Text(''),
+                  )
+                ]))));
+    await tester.pumpAndSettle();
+    final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse, pointer: 1);
+    await gesture.moveTo(tester.getCenter(find.byType(Tab)));
+    await tester.pumpAndSettle();
+    final RenderObject object = tester.allRenderObjects.firstWhere((RenderObject element) => element.runtimeType.toString() == '_RenderInkFeatures');
+    expect( object, paints..rrect(
+                                color: _hoverColor,
+                                rrect: RRect.fromRectAndRadius(
+                                    tester.getRect(find.byType(InkWell)),
+                                    const Radius.circular(_radius))));
```

Comment:
The formatting needs correction. Refer to the Style Guide or other tests to know more.